src/main/java/org/soen343/controllers/DashboardController.java

```diff
@@ -59,9 +82,47 @@ public void initialize() {
         houseLayoutController.update();
     }
 
-    /**
-     * Update
-     */
+    public void updateTime(Calendar calendar) {
+        Date date = calendar.getTime();
+        DateFormat timeFormat = new SimpleDateFormat("hh:mm:ss");
+        DateFormat dateFormat = new SimpleDateFormat("MM/dd/yy");
+        currentTime.setText(timeFormat.format(date));
+        currentDate.setText(dateFormat.format(date));
+    }
+    private void startAnimatedTime() {
+        timer = new Timer();
+        timer.scheduleAtFixedRate(new TimerTask() {
+            @Override
+            public void run() {
+                Platform.runLater(() -> updateTime(SimulationParameters.dateTime.getDate()));
+            }
+        },0, (long) (Duration.ofSeconds(1).toMillis()/SimulationParameters.dateTime.getClockSpeedMultiplier()));
+    }
+
+    private void stopAnimatedTime() {
+        timer.cancel();
+    }
+
+    private void changeAnimatedTime() {
+        stopAnimatedTime();
+        startAnimatedTime();
+    }
+
+    private void addListenerToMultiplierSlider() {
+        slider.valueProperty().addListener(
+                (observableValue, oldValue, newValue) -> {
+                    double multiplier = Math.round((Double) newValue);
+                    SimulationParameters.dateTime.setClockSpeedMultiplier(multiplier);
+                    currentMultiplier.setText(String.valueOf(multiplier));
+                    changeAnimatedTime();
+                }
+        );
+    }
+
+
+                /**
+                 * Update
```

Comment:
All of this code should be in the SimulationInfoController, because it is there that display the clock. 
You will need to change the update part about the old time and insert your new code.